.travis.yml

```diff
@@ -6,8 +6,17 @@ before_script:
   - "npm install"
   - "npm run build"
   - "java -jar selenium-server-standalone-2.53.0.jar -Dphantomjs.binary.path=./node_modules/phantomjs-prebuilt/bin/phantomjs &"
-  - sleep 3
   - npm start &
-  - sleep 5
+  - "chmod 750 waiting-for-start.sh"
```

Comment:
This is not necessary. You could just do `sh waiting-for-start.sh` or `bash waiting-for-start.sh` by your preference.